modules/cnv_template_openshift_inventory.adoc

```diff
@@ -0,0 +1,49 @@
+[[example-inventory-file]]
+=== {product-title} example inventory file
+
+The file below is an example Ansible inventory file for an {product-title} 
+cluster suitable for {ProductName}. In this scenario, the master node is also 
+an infra node, and there are two separate compute nodes. The same three nodes 
+form an OpenShift Container Storage `[glusterfs]` cluster which is used for the 
+storage backend.
+
+----
+[OSEv3:children]
```

Comment:
@ousleyp it seems there's a special character here to remove.

Reply:
I retyped the line and Nelly confirmed it appears to be gone now. Thanks Federico!